### ISSUE TITLE
Screen reader status not validated

### ISSUE THREAD

**Issue by markup-mitchell** [markup-mitchell/digirati-countries#2]
Need to test the app with NVDA. This should be a backlog task rather than an issue, but will track it at least. 

Screen reader performance might give some insight about whether the component root elements are appropriate, so might inform other tasks/tickets.

Would accessible tabs be better for this task? 

Ideally a UX designer or a11y SME could feed in to the decision!
